src/adaXT/predict/predict.pyx

```diff
@@ -175,7 +175,7 @@ cdef class PredictClassification(Predict):
                     cur_node = cur_node.right_child
             if self.classes is not None:
                 ret_val.append(cur_node.value)
-        return np.array(ret_val, dtype=int)
+        return np.array(ret_val)
```

Comment:
For the classification specifically, do we not want to return an integer array?

Reply:
Yes, but this happens already in `__predict` or not?

Reply:
I take this back. It should never just be int. One might have a classification, where the outcome is a float of some kind.